elementary/monitor/api/alerts.py

```diff
@@ -19,27 +19,29 @@ def __init__(self, dbt_runner: DbtRunner, elementary_database_and_schema: str):
         super().__init__(dbt_runner)
         self.elementary_database_and_schema = elementary_database_and_schema
 
-    def query(self, days_back: int) -> Alerts:
+    def query(self, days_back: int, timezone: Optional[str] = None) -> Alerts:
```

Comment:
All the changes in the `query*` functions are not necessary.

Reply:
removed it all